docs/api/datasets/non_geo_datasets.csv

```diff
@@ -15,6 +15,7 @@ Dataset,Task,Source,License,# Samples,# Classes,Size (px),Resolution (m),Bands
 `ETCI2021 Flood Detection`_,S,Sentinel-1,-,"66,810",2,256x256,5--20,SAR
 `EuroSAT`_,C,Sentinel-2,"MIT","27,000",10,64x64,10,MSI
 `FAIR1M`_,OD,Gaofen/Google Earth,"CC-BY-NC-SA-3.0","15,000",37,"1,024x1,024",0.3--0.8,RGB
+`Fields Of The World`_,"S,I",Sentinel-2,"Various","70795","2,3",256x256,10,MSI
```

Comment:
The labels for each country dataset have a potentially different license. If you click through the links in the table shown https://beta.source.coop/repositories/kerner-lab/fields-of-the-world/ you can see the individual licenses. E.g. Austria is CC-BY-4.0 (https://beta.source.coop/repositories/kerner-lab/fields-of-the-world-austria/description/) while South Africa is CC-BY-NC-SA-4.0. The point of having the dataset download data per-country is so that the end-user can easily choose what data they want to use.

Reply:
FYI we will update https://beta.source.coop/repositories/kerner-lab/fields-of-the-world/ with the license for each country in the dataset, and I'll copy this information over into the constructor docstring